package solidexercicio10.model;

public class Engenheiro {
    public Engenheiro(String nome, int x, int y) {
        super(nome, "Engenheiro", x, y);
    }

    @Override
    public int getPontuacao() {
        return 15;  // Engenheiro vale 15 pontos
    }
}
